### PR TITLE
Release-note-update-7_6_4_search

### PR DESCRIPTION
**Pull request by sarahlwelton**
Missing a feature update for 7.6.4, just sliding this in here.

### CODE REVIEW

File: modules/introduction/partials/new_features-76_4.adoc [hunk 1]
```diff
@@ -33,17 +33,14 @@ curl --get -u <username:password> \
...
 
+* The Search Service now supports a new option for optimizing Vector Search indexes: *memory-efficient*.
+Choose this option to prioritize reduced memory and resource usage for Vector Searches, at the cost of accuracy and latency. 
+For more information, see xref:search:child-field-options-reference.adoc[Child Field Options].
+
 * The Search Service has added a new object to JSON Search queries. 
 Use this new object to view detailed query debugging information and resolve query errors in the Web Console or through the REST API.
 For more information about how to run a query with this new object, see xref:search:search-request-params.adoc#validate[the validate property], xref:search:simple-search-rest-api.adoc#example-validate-a-search-query[Run a Simple Search with the REST API and curl/HTTP] or xref:search:simple-search-ui.adoc#example-validate-a-search-query[Run A Simple Search with the Web Console].
 
-[#new-features-764-eventing-service]
```
Comment (simon-dew) [inline modules/introduction/partials/new_features-76_4.adoc]: Can you restore this section please? Thanks

**Review by simon-dew — CHANGES_REQUESTED** [review]
Looks like a section got missed

File: modules/introduction/partials/new_features-76_4.adoc (hunk 1 above)
Comment (sarahlwelton) [reply modules/introduction/partials/new_features-76_4.adoc]: Ah, must have been a bad merge, one sec

**Review by simon-dew — APPROVED** [review]
LGTM 👍

**Pull request merged**